src/entity/Family.ts

```
@@ -62,12 +62,6 @@ export class Family implements FamilyInterface {
   @OneToMany(() => IncomeDetermination, (det) => det.family)
   @ValidateNested({ each: true })
   @ArrayMinSize(1)
-  @ValidateIf((family) => {
```

There are some other places that need to be updated for this change-- off the top of my head, I think the fake data creation (maybe?) and a front end func that lives with the family income form and determines if it has errors.  Also the family income forms that skip past if the child is foster.

---

I think I caught the part of the front-end func that determines errors, as well as not skipping the income form in the create flow for a foster child. I'm not seeing anything in the fake data creation that breaks though. Which part do you think the bug lives in?

---

Ah I see what you mean in the create section--we still want it to show as complete and not exempt. Fixed it this time!